src/HttpRequest.cc

```diff
@@ -368,9 +368,7 @@ httpRequestPack(void *obj, Packable *p)
 int
 HttpRequest::prefixLen() const
 {
-    return method.image().length() + 1 +
-           url.path().length() + 1 +
-           4 + 1 + 3 + 2 +
+    return Http::One::RequestParser::RequestLineSize(method.image().length(), url.path().length()) +
```

Comment:
This change preserves the symmetry between request and response handling code. It also reduces code duplication. The magical-looking math is preserved in this transition to Http::One::RequestParser::RequestLineSize().